ui/src/app/applications/components/application-operation-state/application-operation-state.tsx

```diff
@@ -84,6 +84,13 @@ export const ApplicationOperationState: React.StatelessComponent<Props> = ({appl
         }
     }
 
+    const [filters, setFilters] = React.useState([]);
+    const filterOptions = ['Synced', 'OutOfSync', 'Unknown'];
```

Comment:
Sync result status could be either one of hook phases or sync phases:


https://github.com/argoproj/gitops-engine/blob/aae8ded161136ccc01cf5f21a99815a15ec2410f/pkg/sync/common/types.go#L46. 

https://github.com/argoproj/gitops-engine/blob/aae8ded161136ccc01cf5f21a99815a15ec2410f/pkg/sync/common/types.go#L32